### PR TITLE
correct various spelling errors in README

### PR DESCRIPTION
**Pull request by cmarkle**
like it says - fix spelling errors

### CODE REVIEW

**Comment by cmarkle** [comment]
OK @gegles let's see if this is better...

File: docs/README.erb.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ Then, follow the section relative to the product you want to interact with ( Asp
 
 # <a name="installation"></a>Installation
 
-It is possible to install *either* directly on the host operating system (Linux, Windows, Macos) or as a docker container.
+It is possible to install *either* directly on the host operating system (Linux, Windows, MacOS) or as a docker container.
```
Comment (gegles) [inline docs/README.erb.md:152]:
```suggestion
It is possible to install *either* directly on the host operating system (Linux, Windows, macOS) or as a docker container.
```
Technically it's now `macOS` ;-)

File: docs/README.erb.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ Required Ruby version: <%=gemspec.required_ruby_version%>. Ruby version 3 is als
 
 Refer to the following sections for a proposed method for specific operating systems.
 
-The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, Macos, Windows with cygwin, etc...).
+The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, MacOS, Windows with cygwin, etc...).
```
Comment (gegles) [inline docs/README.erb.md:212]:
```suggestion
The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, macOS, Windows with cygwin, etc...).
```

File: README.md [hunk 3]
```diff
@@ -148,7 +148,7 @@ Then, follow the section relative to the product you want to interact with ( Asp
 
 # <a name="installation"></a>Installation
 
-It is possible to install *either* directly on the host operating system (Linux, Windows, Macos) or as a docker container.
+It is possible to install *either* directly on the host operating system (Linux, Windows, MacOS) or as a docker container.
```
Comment (gegles) [inline README.md:151]:
```suggestion
It is possible to install *either* directly on the host operating system (Linux, Windows, macOS) or as a docker container.
```
Technically it's now macOS ;-)

File: README.md [hunk 4]
```diff
@@ -208,7 +208,7 @@ Required Ruby version: > 2.4. Ruby version 3 is also supported.
 
 Refer to the following sections for a proposed method for specific operating systems.
 
-The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, Macos, Windows with cygwin, etc...).
+The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, MacOS, Windows with cygwin, etc...).
```
Comment (gegles) [inline README.md:211]:
```suggestion
The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, macOS, Windows with cygwin, etc...).
```

**Review by gegles — APPROVED** [review]
This looks great @cmarkle, thanks!

I think you can simply take my suggestions from the webapp (if you want). Click 'add to batch'

File: README.md (hunk 4 above)
Comment (cmarkle) [reply README.md]: I agree. I made another commit for this and pushed it. Can I get that added to this pull request or do I make another pull request?

**Review by gegles — APPROVED** [review]
Perfect, thanks @cmarkle!

**Pull request merged**